cypress/e2e/ModelizerPage/SwitchView/delete.feature

```diff
@@ -0,0 +1,227 @@
+Feature: Test switch model to text view: delete component/link
+
+  Background:
+    Given I clear cache
+    And I set viewport size to "1536" px for width and "960" px for height
+    And I visit the "/"
+
+    When I click on "[data-cy=\"new-project\"]"
+    Then I expect current url is "/modelizer/project-[a-f0-9]{8}/model"
+    And  I extract "project-[a-f0-9]{8}" from url in field "projectName" of context
+    And  I expect "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"]" exists
+    And  I expect "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"] [data-cy=\"plugin-definitions-title\"]" is "leto-modelizer-plugin-test"
+    
+    When I click on "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"]"
+    Then I expect ".plugin-definitions" appear 1 time on screen
+    And  I expect ".component-definition-card" appear 7 times on screen
+
+  # Scenario: Delete a component (Model view) should remove corresponding object inside plugin file content (Text view)
```

Comment:
to delete

Reply:
done

Reply:
keep test but added `@skip` tag to not run in with Cypress 
https://github.com/badeball/cypress-cucumber-preprocessor/pull/100